wgsl/index.bs

```diff
@@ -1292,6 +1292,25 @@ const_expr
   </xmp>
 </div>
 
+## Return statement ## {#return-statement}
+
+<pre>
+  | RETURN logical_or_expression? SEMICOLON
+</pre>
+
+A `return` statement ends execution of the current function.
+If the function is an entry point, then the current shader invocation
+is terminated.
+Otherwise, evaluation continues with the next expression or statement after
+the evaluation of the call site of the current function invocation.
+
+If the function returns the void type, then the return statement must not have
+an expression.
+
```

Comment:
I think you want to either remove this blank line, or add one before 1304 ? Otherwise, I think this will end up as two paragraphs but the one above will be one paragraph? Maybe run the `make online` command and take a look at the resulting html file to see if it renders correctly.